README.adoc

```diff
@@ -112,7 +111,7 @@ https://lists.jboss.org/mailman/listinfo/hawkular-dev[Hawkular developer mailing
 
 We love contributions and pull-requests :-)
 
-To meet the guys, join us on irc://irc.freenode.net/#hawkular[Freenode IRC] in channel #hawkular.
+To chat, join us on irc://irc.freenode.net/#hawkular[Freenode IRC] in channel #hawkular.
```

Comment:
If possible, instead of me creating one more PR about these changes, could you modify this PR to include the following changes also:

-It can also be used independently.
+It can also be used independently or with https://github.com/hawkular/hawkular-alerts[Alerts].

And to contribute part:

+To file an issue, use the http://issues.jboss.org/browse/HWKMETRICS[Hawkular-Metrics JIRA].

(this was actually asked in the IRC, where to report bugs and we don't have a link anywhere)